test/preset-attributify.test.ts

```diff
@@ -128,6 +128,11 @@ describe('attributify', () => {
       ]}
     >`
 
+  const fixture6 = `<div foo={\`\${5}\`} foo={"\${5}"} foo={'\${5}'} >
+  \`'"
+  <div bg="red" >
+  `
```

Comment:
There's another backtick at the end as part of user string. Without the backtick, the bg-red token is properly generated. Maybe we could add more test with both test samples.

Reply:
I tested it on the playground, it works as expected. The first line of fixture already covers those situations.